### PR TITLE
pdf attachments attributes: https://github.com/metanorma/mn2pdf/issue…

### PR DESCRIPTION
**Pull request by opoudjis**
…s/278

### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

File: author/topics/collections/configuration.adoc [hunk 1]
```diff
@@ -171,6 +171,10 @@ coverpage and boilerplate) [added in https://github.com/metanorma/metanorma/rele
 This is automatically included when `sectionsplit` is set in the
 Metanorma file, to break a single document up into multiple HTML files.
 
+`flavor`:::
```
Comment (ronaldtse) [inline author/topics/collections/configuration.adoc:174]: In the document we call a flavor `:mn-document-class: iso`. Can we use a single nomenclature for this?

Comment (ronaldtse) [reply author/topics/collections/configuration.adoc:174]: We should accept the same names both in the collection config and as document attribute. We could either call it "document-class" or accept both "flavor" and "document-class".

Comment (opoudjis) [reply author/topics/collections/configuration.adoc:174]: I'll go with the latter; document-class is, in retrospect, too ambiguous to recommend.

Comment (ronaldtse) [reply author/topics/collections/configuration.adoc:174]: I agree. And maybe we should get rid of the "mn-" prefix one day too...

Comment (ronaldtse) [reply author/topics/collections/configuration.adoc:174]: I'm going to merge this one first, do the document attribute rename of "document-class" to "flavor" in a separate PR.

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**